.github/workflows/ci.yml

```diff
@@ -0,0 +1,152 @@
+        run: yarn install --immutable --immutable-cache
+      - name: 📈 Run GraphQL Codegen
+        run: |
+          yarn gql
+          yarn gql:server
+      - name: 🧪 Test (chunk ${{ matrix.chunk }})
+        run: echo $CHUNKS | jq '.[${{ matrix.chunk }}] | .[] | @text' | xargs yarn test:coverage --ci
+        env:
+          CHUNKS: ${{ needs.split-tests.outputs['test-chunks'] }}
+      - name: Codecov
+        run: bash <(curl -s https://codecov.io/bash)
+
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 15
+      - name: 📦 Install Dependencies
+        run: yarn install --immutable --immutable-cache
+      - name: 📈 Run GraphQL Codegen
+        run: |
+          yarn gql
+          yarn gql:server
+      - name: 💨 ESLint
+        run: yarn lint:ci
+
+  typescript:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 15
+      - name: 📦 Install Dependencies
+        run: yarn install --immutable --immutable-cache
+      - name: 📈 Run GraphQL Codegen
+        run: |
+          yarn gql
+          yarn gql:server
+      - name: ✅ TypeScript lint
+        run: yarn lint:ts
+
+  prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 15
+      - name: 📦 Install Dependencies
+        run: yarn install --immutable --immutable-cache
+      - name: 💅 Prettier Check
+        run: yarn prettier:check
+
+  chromatic:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Chromatic needs access to the full Git history graph
+          fetch-depth: 0
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 15
+      - name: 📦 Install Dependencies
+        run: yarn install --immutable --immutable-cache
+      - name: 📈 Run GraphQL Codegen
+        run: |
+          yarn gql
+          yarn gql:server
+      - name: 📙 Upload to Chromatic
+        if: ${{ github.event_name == 'pull_request' }}
+        run: yarn chromatic --exit-zero-on-changes --skip 'dependabot/**' --exit-once-uploaded
+        env:
+          CHROMATIC_PROJECT_TOKEN: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+      - name: 📗 Upload to Chromatic accepting changes
+        if: ${{ github.event_name != 'pull_request' }}
+        run: yarn chromatic --auto-accept-changes --skip 'dependabot/**' --exit-once-uploaded
+        env:
+          CHROMATIC_PROJECT_TOKEN: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+  yarn-check-cache:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 15
+      - name: 📦 Install Dependencies
+        run: yarn install --immutable --immutable-cache --check-cache
+
+  # onesky-upload:
```

Comment:
Are we waiting for Onesky stuff until this is in prod? Or were there issues getting this to work?

Reply:
Nothing has been configured yet. Idk if this repo even has a OneSky project. Maybe I should have left this out until we were ready. But I decided just to copy it over so we have it.